README.md

```diff
@@ -3,32 +3,32 @@
 This module containerizes and provides interfaces for the backend of ToyNet to
 interface with the Mininet emulator implementing the topology that a user builds.
 
-## Building
+## Building and Running
 
-Run `docker build -t toynet .` to build the production image using the `Dockerfile`.
-For the development image, run `docker build -t toynet-dev -f dev.Dockerfile .`
+Building and Running are controlled by the `Makefile`, run `make help` to identify
+the appropriate target for your use-case.
+
+There are targets for production and test use-cases; it is recommended to use the 
+test series of targets for development.
 
 It will take several minutes the first time for both images, but after the
 majority of the images get cached by Docker it will be much faster.
 
-## Running
-
-Assuming you tagged the images as specified above:
-
-To run the production image:
-run `docker run --privileged -v sample.xml:/root/toynet-mininet/topo.xml -v /lib/modules:/lib/modules toynet`
-
-And to run the development image:
-run `docker run --privileged -v sample.xml:/root/toynet-mininet/topo.xml -v /lib/modules:/lib/modules toynet-dev`
+# Installing Dependencies
 
-# Python Library
 This library makes some functionality of the Docker Python API more readily
 accessible for your use case. This library is intended to facilitate Flask
 managing Mininet containers local to its running instance.
 
-## Installing Dependencies
+## External Repositories
```

Comment:
Does the linting of 80 character limit apply to .md files? Perhaps they should not so we can have paragraphs in READMEs.